client/src/community/contributor-list.js

```diff
@@ -1,18 +1,15 @@
 import { LitElement, html } from "lit";
-import { customElement } from "lit/decorators.js";
 
 import styles from "./contributor-list.scss?css" with { type: "css" };
 
-interface ContributorData {
-  name: string;
-  github: string;
-  org?: string;
-}
+/**
+ * @typedef {Object} ContributorData
+ * @property {string} name
+ * @property {string} github
+ * @property {string} [org]
+ */
```

Comment:
My preference would be to define these in a `.d.ts` file next to the JavaScript, as it's a bit easier / less verbose + error-prone than the JSDoc. Wdyt?

Reply:
Yeah, good idea: it's a pattern we'll be using a lot anyway with rari emitting typescript types (perhaps it could be configured to also emit jsdoc types? but it'll be easier for migration if we keep all the types in typescript, at least for now)